src/ElementalChat.vue

```diff
@@ -215,17 +215,33 @@ export default {
     handleToDisplay () {
       return this.isHoloAnonymous ? 'anonymous' : this.agentHandle
     },
+    shouldHandleLogin() {
+      return isHoloHosted() && this.holoStatus === 'ready' && this.isHoloAnonymous === true
+    },
     canMakeZomeCalls() {
-      return isHoloHosted() ? this.holoStatus === 'ready' : !this.conductorDisconnected
+      return isHoloHosted() ? this.holoStatus === 'ready' && this.isHoloAnonymous === false : !this.conductorDisconnected
```

Comment:
Develop includes anonymous by default. This change makes the anonymous use case not load all the messages at the start. I suggest reverting this line, and checking for anonymous in `canMakeZomeCalls` watcher if necessary